app/logic/emailHandler.py

```diff
@@ -169,7 +171,7 @@ def send_email(self):
                         reply_to=self.reply_to,
                         sender = ("Sandesh", 'bramsayr@gmail.com')
                     ))
-            self.store_sent_email(subject, template_id)
+            self.store_sent_email(subject, template_id, g.current_user)

```

Comment:
We don't want email handler to be dependent on the application session. Add an initialization argument for user and use that here. Then the calling code can pass in `g.current_user`.